src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,8 +1,60 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
+    private static final int ACTION_INDEX = 0;
+    private static final int VALUE_INDEX = 1;
 
     public void getStatistic(String fromFileName, String toFileName) {
+        String dataFromFile = readFromFile(fromFileName);
+        String report = createReport(dataFromFile);
+        writeToFile(toFileName, report);
+    }
+
+    private String readFromFile(String fromFileName) {
+        StringBuilder firstFileData = new StringBuilder();
+        try (BufferedReader br = new BufferedReader(new FileReader(fromFileName))) {
+            String value = br.readLine();
```

Comment:
```suggestion
            String line = br.readLine();
```
'value' is not informative name.